app/web/src/features/communities/events/EditEventPage.tsx

```diff
@@ -112,23 +113,26 @@ export default function EditEventPage() {
     ) : isEventLoading ? (
       <CircularProgress />
     ) : (
-      <EventForm
-        error={error}
-        event={event}
-        isMutationLoading={isLoading}
-        mutate={updateEvent}
-        title="Edit event"
-      >
-        {({ isMutationLoading }) => (
-          <Button
-            className={classes.submitButton}
-            loading={isMutationLoading}
-            type="submit"
-          >
-            {UPDATE}
-          </Button>
-        )}
-      </EventForm>
+      <>
+        <HtmlMeta title="Edit event" />
+        <EventForm
+          error={error}
+          event={event}
+          isMutationLoading={isLoading}
+          mutate={updateEvent}
+          title="Edit event"
```

Comment:
Move "Edit event" to a constant please (my bad not doing it in the first place...)

Reply:
done